templates/email/default/other-reported.html

```diff
@@ -12,14 +12,22 @@
   [% start_padded_box | safe %]
   <h1 style="[% h1_style %]">Your report has been&nbsp;logged</h1>
   <p style="[% p_style %]">Your report to [% report.body %] has been logged on [% site_name %].</p>
+
+[% IF report.non_public %]
+<p style="[% p_style %]">It has been marked as private and will not be visible
+to the general public; you may view it using the link below, or if you sign in
+using the email address associated with the report.</p>
```

Comment:
This link stops working after a month. We should a) indicate this to the user and b) make it clear that after a month they will *need* to login with their email address to view the report.

Reply:
At that point, the website does say "If you are the problem reporter, or a member of staff, please <a href="%s">sign in</a> to view this report." which I thought covered it well enough?

Reply:
The message shown is actually: 
<img width="589" alt="image" src="https://user-images.githubusercontent.com/4776/74323608-af913000-4d7d-11ea-92bf-02d80c5d2b03.png">

I can imagine a scenario where a user refers back to the email each time they want to view the report, so it'd be good to manage their expectations around the time limited nature of the link.

Reply:
Ah, right, sorry, yep, perhaps it shouldn't expire the token (given it doesn't log you in). Or (easier) not have a token at all, scrub that bit of code, and always just give the "please log in" page. That would fix the leaving an update issue as well.